tests/cpp/common/test_device_helpers.cu

```diff
@@ -96,84 +96,6 @@ TEST(bulkAllocator, Test) {
   TestAllocator();
 }
 
-template <typename T, typename Comp = thrust::less<T>>
```

Comment:
have we touched this test?

Reply:
I do not know, but it shoud keep the same with our internal branch